service/src/main/java/uk/gov/hmcts/reform/fpl/service/ChildrenMigrationService.java

```diff
@@ -0,0 +1,48 @@
+package uk.gov.hmcts.reform.fpl.service;
+
+import com.google.common.collect.ImmutableMap;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+@Service
+public class ChildrenMigrationService {
+    private static final String MIGRATED_CHILDREN_KEY = "children1";
```

Comment:
is this necessary?

Reply:
No it's not, sonar cloud was complaining. But I'll revert this.